### PR TITLE
Added tensorboard

### PR DESCRIPTION
**Pull request by Tekrus**

### CODE REVIEW

File: model/Graph-conv-context-aware-idea-2021-wip/model.py [hunk 1]
```diff
@@ -198,21 +241,54 @@ def _partial_fit(self, data):
         feed_dict = {self.users: data['user_ids'], self.pos_interactions: data['pos_interactions'],
                      self.neg_interactions: data['neg_interactions'], self.context: data['contexts'],
                      self.user_sideinfo: data['user_sideinfo'], self.item_sideinfo: data['item_sideinfo']}
-        return self.sess.run([self.opt, self.loss], feed_dict=feed_dict)
+        return self.sess.run([self.loss, self.opt], feed_dict=feed_dict)
 
     def train(self):
+        # tennsorboard
```
Comment (Eduardsen) [inline model/Graph-conv-context-aware-idea-2021-wip/model.py]:
```suggestion
        # Tensorboard
```

Comment (Eduardsen) [reply model/Graph-conv-context-aware-idea-2021-wip/model.py]: xd

Comment (Tekrus) [reply model/Graph-conv-context-aware-idea-2021-wip/model.py]: teñsõr

**Review by Eduardsen — APPROVED** [review]

File: model/Graph-conv-context-aware-idea-2021-wip/model.py [hunk 2]
```diff
@@ -112,8 +156,7 @@ def _init_graph(self):
         self.neg_scores = tf.nn.dropout(self.neg_scores, args.keep_prob)
 
         self.loss = self._bpr_loss(self.pos_scores, self.neg_scores)
-        self.opt = tf.train.AdamOptimizer(
-            self.learning_rate).minimize(self.loss)
+        self.opt = self.optimizer.minimize(self.loss[0])
```
Comment (andpaacoke) [inline model/Graph-conv-context-aware-idea-2021-wip/model.py:159]: loss[0]? hvad gør det?
